extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -251,6 +251,15 @@ export const externalStreamingJobFriendlyName = localize('externalStreamingJobFr
 export const preDeployScriptFriendlyName = localize('preDeployScriptFriendlyName', "Script.PreDeployment");
 export const postDeployScriptFriendlyName = localize('postDeployScriptFriendlyName', "Script.PostDeployment");
 
+// Build
+
+export const NetCoreInstallationConfirmation: string = localize('sqlDatabaseProjects.NetCoreInstallationConfirmation', "The .NET Core SDK cannot be located. Project build will not work. Please install .NET Core SDK version 3.1 or update the .Net Core SDK location in settings if already installed.");
```

Comment:
nit: update the last .Net to be .NET like the previous 2 (also in the string below)